.wuf/generator/templates/desktop/Component/Component.st.css

```diff
@@ -1,18 +1,30 @@
 :import {
     -st-from: "../../common/formatters.st";
-    -st-named: color;
+    -st-named: color, fallback;
 }
 
-/*Defaults*/
+/* Defaults - default params */
 :vars {
-    /*Defaults params goes here*/
+  defaultBgColor: color(color-1);
 }
 
-/*Overrides*/
+/* Overrides - deceleration of overridable parameters by the consumer */
 :vars {
-    /*Deceleration of overridable placeholders*/
+  /*
+    Main background color
+  */
+  BgColor: --overridable;
```

Comment:
Do you maybe want to use the autodocs Stylable plugin "syntax" of adding defaults?
https://github.com/wix/wix-ui-tpa/blob/48aa71d1a6cd07f139561d4b559a80754e72977c/src/components/TextArea/TextArea.st.css#L51

And maybe we don't even need to split the `defaults` and the `overridables`?
Just have it:
```css
  /*
    Main background color
    @default color-1
  */
  BgColor: color(color-1);
```

Reply:
I'll need to experiment it further so I'm keeping it out of this current PR